packages/cli-dashboard/src/components/utils/tests/reshapeCookies.ts

```diff
@@ -31,7 +31,7 @@ describe('reshapeCookies', () => {
   it('should return an object with the cookies', () => {
     const cookies = {
       'https://edition.cnn.com': {
-        'countryCode:.cnn.com:/': {
+        'countryCode.cnn.com:/': {
```

Comment:
Why did the key formating change?

Reply:
It was to fix the test but that has been resolved now in latest commits.